### DISCUSSION TITLE
(Coreshop Datahub) Assigning the coreshop_coreshop_relation to an object

### DISCUSSION THREAD

**Discussion by twin-elements** [coreshop/CoreShop#2597] [Q&A] [answered]
Using the DataHub importer (Load DataObject transformation), I load a CoreShopManufacturer object and need to assign it to a product.

![image](https://github.com/coreshop/CoreShop/assets/17727266/f2959f83-b0c7-4cff-9dde-22f29403cb42)

But in the data Target it is not possible to assign a manufacturer (not available).

![image](https://github.com/coreshop/CoreShop/assets/17727266/7390dd28-267f-4577-80d7-29be1ad89505)

Is it possible to assign a coreshop_coreshop_relation to an object?

**Comment by dpfaffenbauer** [comment, marked as answer]
Hi,

same issue as with the Prices. The CoreShop Relation type inherits from the Pimcore Relation Type, but for Pimcore it still is a new Data Type. Therefore we would need to implement the data-type in Datahub Importer. We usually don't use DataHub Importer but DataDefinitions instead (https://github.com/instride-ch/datadefinitions)
